packages/amplify-codegen/src/commands/types.js

```diff
@@ -93,32 +93,25 @@ async function generateTypes(context, forceDownloadSchema, withoutInit = false,
         codeGenSpinner.start();
         const schema = fs.readFileSync(schemaPath, 'utf8');
         const introspection = path.extname(schemaPath) === '.json';
-
+        const multipleSwiftFiles = target === 'swift' && fs.existsSync(outputPath) && fs.statSync(outputPath).isDirectory();
```

Comment:
If I'm running for the first time will `outputPath` necessarily exist? Do we instead need to rely on the path filename suffix or something like that?

Reply:
Ahh, I just saw that's the old behavior as well.